### PR TITLE
Publish latest templates

### PR DESCRIPTION
**Pull request by joebochill**
#### Changes proposed in this Pull Request:
- Update dependencies used by the templates
- 
Waiting for #14

### CODE REVIEW

File: blank-typescript/template.json [hunk 1]
```diff
@@ -3,36 +3,34 @@
...
             "@material-ui/icons": "^4.9.1",
-            "@pxblue/react-themes": "^6.0.0-beta.0",
-            "@pxblue/react-components": "^4.1.1",
-            "@pxblue/icons-mui": "^2.2.1",
+            "@pxblue/react-themes": "^6.0.0",
+            "@pxblue/react-components": "^5.1.0",
+            "@pxblue/icons-mui": "^2.3.0",
             "@pxblue/colors": "^3.0.0",
             "react-app-polyfill": "latest",
             "typescript": "^4.0.3",
```
Comment (huayunh) [inline blank-typescript/template.json:11]: Maybe this answers my question why we are having typescript in every single re-scaffolding PRs

Comment (joebochill) [reply blank-typescript/template.json:11]: Yes, this is where it comes from. It's based on the default create-react-app template, which puts typescript into dependencies.

Comment (joebochill) [reply blank-typescript/template.json:11]: If we want to move it we can, I just have to check if that will end up giving us two references to typescript or not.

Comment (huayunh) [reply blank-typescript/template.json:11]: #17

**Review by huayunh — APPROVED** [review]

**Review by jeffvg — APPROVED** [review]
looks good

**Pull request merged**